### DISCUSSION TITLE
How to "reset" the cluster using K3s?

### DISCUSSION THREAD

**Discussion by berviantoleo** [k3s-io/k3s#7208] [Q&A] [answered]
Hi, I'm new to K3s.

I want to reset the cluster. How to do that? Do I need to uninstall the K3s script and install the K3s again? Do we have commands like k3s reset or k3s delete?

**Comment by brandond** [comment, marked as answer]
Uninstall and reinstall is the best way to ensure you have a clean node. You can also just run the killall script and then `rm -rf /var/lib/rancher/k3s` and that should take care of most things.

You should do this on all nodes in the cluster before starting things up again.
